docs/user/guides/custom-domains.rst

```diff
@@ -0,0 +1,121 @@
+How To Manage Custom Domains
+============================
+
+In this guide, you will find the simple steps of how to host your documentation from your own domain name, such as ``yourdomain.tld`` or ``docs.yourdomain.tld``.
```

Comment:
Except for addition of the introduction, none of the original text is modified.

Reply:
It's best to always use `example.com` for mock domains, as it's a reserved name:
https://www.iana.org/domains/reserved

Apex domains are probably worth avoiding in the intro as well, as we only support this on certain providers. The various implementations of cname flattening/aname/etc are all proprietary features.

Also, a few less words for clarity:

```suggestion
This guide describes how to host your documentation using your own domain name, such as  ``docs.example.com``.
```

Reply:
Sounds good with the domain :+1:

I don't have preferences regarding active or passive language. It's nice to be consistent of course! But I think we are mixing the two already? Anyways, I like both the conciseness or accuracy that passive language often leads to... but active language is often friendlier and easier to read.

I thought the introduction paragraph was a good way to introduce the subject with an active voice and then try to adapt the texts later.

Reply:
Btw our guidelines don't say anything about active/passive nor writing style in general: https://dev.readthedocs.io/en/latest/docs.html

Is there another resource I've been missing?

Reply:
We don't have an internal styleguide, this is more general advice for documentation -- write clear, concise sentences. The MS styleguide is fairly thorough and modern, I defer to that usually.

https://learn.microsoft.com/en-us/style-guide/word-choice/use-simple-words-concise-sentences

I'm not sure I follow on your point about active and passive language. What are you describing here?

_Active voice_, is something to aim for, but neither of these sentences speak in passive voice. The original text just wasn't as definitive and confident about the purpose of this guide -- "In this guide, you will find X" instead of "This guide has X". I suppose for that, I'd say the original sounds more passive.

Reply:
There must be another term then. I'm referring to putting "you", i.e. the reader, into sentences. I have previously written in Danish where the styleguide said to use "you".

Reply:
That's a good precise guide!

I also like [Hemingway](https://hemingwayapp.com/) and I like the writings by Adam Johnson:

https://adamj.eu/tech/2019/05/06/improving-my-technical-writing/

Reply:
> I'm referring to putting "you", i.e. the reader, into sentences.

Ah gotcha! No, this isn't a problem at all. Speaking to the user directly is still absolutely fine, especially if we are instructing the user to take some action. It just resulted in less clear language in the original text.

Reply:
This was good feedback and nice to re-read. I will mark it as resolved :+1: